templates/artist/includes/artist_detail_projects.html

```diff
@@ -0,0 +1,25 @@
+{% load staticfiles %}
+{% load humanize %}
+{% load thumbnail %}
+{% load markdown_deux_tags %}
+{% load perdiem %}
+{% load music %}
+
+<div class="tabs-panel" id="projects">
+    {% if campaign %}
```

Comment:
If the artist isn't currently running a campaign, this tab will be completely blank. Is that okay?